src/main/java/com/amazon/opendistroforelasticsearch/jdbc/protocol/http/JdbcCursorQueryRequest.java

```diff
@@ -0,0 +1,59 @@
+package com.amazon.opendistroforelasticsearch.jdbc.protocol.http;
+
+import com.amazon.opendistroforelasticsearch.jdbc.protocol.JdbcQueryParam;;
+import com.amazon.opendistroforelasticsearch.jdbc.protocol.QueryRequest;
+
+
+import java.util.List;
+import java.util.Objects;
+
+public class JdbcCursorQueryRequest implements QueryRequest {
```

Comment:
What is relationship between JdbcRequest and JdbcCursorQueryRequest? The code is almost same, only difference is cursor.

Reply:
No relationship. `JdbcCursorQueryRequest` encapsulates `cursorId`, while `JsonQueryRequest` encapsulates `fetchSize` and `SQL query`.